engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -797,7 +798,7 @@ private void execute() throws YarnException, IOException
       }
 
       numTotalContainers += containerRequests.size();
```

Comment:
numTotalContainers is only used in diagnostics/log output and not used in any decision making. Also it is incremented only here (but never decremented anywhere). This was old code and doesn't affect this JIRA so I am going to leave it unchanged.